test/service/test_response.py

```diff
@@ -2204,6 +2204,31 @@ def test_version(self):
                             }
                             self.assertEqual(expected_json, response.json()['git'])
 
+    def test_response_error_escaping(self):
+        unescaped = json.dumps({
+            'Code': 'NotFoundError',
+            'Message': "Unable to find file 'foo', version None in catalog 'test'"
+        }, separators=(',', ':'))
+        escaped = (
+            "<html><head>Status 404</head><body><pre>{"
+            "'Code': 'NotFoundError', "
+            "'Message': \"Unable to find file 'foo', version None in catalog 'test'\""
+            "}</pre></body></html>"
+        )
```

Comment:
This is all very hard to parse and understand. Consider triple-quoted string and textwrap.dedent.

Also don't mix serialized and deserialized literals. Makes it even harder to examine for correctness.